.lagoon.yml

```diff
@@ -122,3 +122,8 @@ environments:
         schedule: "M/30 * * * *"
         command: drush dpl_po:import-remote-config-po da https://danskernesdigitalebibliotek.github.io/dpl-cms/translations/da.config.po
         service: cli
+    routes:
+      - nginx:
+          - reload.dresscoder.dk:
+              tls-acme: "true"
+              insecure: Redirect
```

Comment:
Skal dette med videre eller er det test indstillinger?

Reply:
You are right, this should be removed, and is removed now!